src/parser.js

```diff
@@ -1489,6 +1595,25 @@ export class Parser extends Tokenizer {
     if (token instanceof IdentifierLikeToken) {
       return new Shift.StaticPropertyName(this.parseIdentifier().name);
     }
+    if (token.type == TokenType.LBRACK) {
+      let previousGeneratorParameter = this.paramGeneratorParameter;
+      let previousYield = this.paramYield;
+      let previousIn = this.paramIn;
```

Comment:
That's not possible. When `MethodDefinition` is reached `paramIn` is always true.